IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerResponseHandler.m

```diff
@@ -139,6 +140,14 @@ - (MSIDBrokerResponse *)brokerResponseFromEncryptedQueryParams:(NSDictionary *)e
     if ([NSString msidIsStringNilOrBlank:decryptedResponse[@"broker_error_domain"]]
         && [decryptedResponse[@"success"] boolValue])
     {
+        // Add redirectUri from resume state only if not present in broker response
+        if (redirectUri && [NSString msidIsStringNilOrBlank:decryptedResponse[MSID_OAUTH2_REDIRECT_URI]])
```

Comment:
Can you clarify why this is needed?

Reply:
Currently, when using nested auth with broker (no SSO extension), the redirectUri is only added to the access token saved by the broker, but not to the AT saved by MSAL app.